docs/topics/minishift-install-nexus-setup.adoc

```diff
@@ -121,55 +124,28 @@ Nexus comes pre-configured for the Maven Central Repository, but you may need ot
 oc project {project-name}
 --
 +
-. Assign the `admin` role to your Jenkins application and the `edit` permissions to your OpenShift account.
-Skip this step if you have already assigned these permissions.
-+
-[source,bash,subs="attributes+"]
---
-oc policy add-role-to-user edit system:serviceaccount:{project-name}:default
-oc policy add-role-to-user admin system:serviceaccount:{project-name}:jenkins
---
-+
-. Obtain the Jenkins server URL and use it to access the Jenkins instance from your browser.
+. Define and start a new build. Pass in a parameter to set the value of the `MAVEN_MIRROR_URL` to match the route of your Nexus application:
 +
-[source,bash,subs="attributes+"]
+[NOTE]
 --
-oc get routes jenkins
+Ensure that the YAML template of the builder image you are using for your application has the `MAVEN_MIRROR_URL` environmental variable defined.
+If it does not, see the Nexus documentation for instructions on link:https://help.sonatype.com/display/NXRM2/Maven+and+Other+Build+Tools[configuring your build manually^] before proceeding.
 --
 +
 [source,bash,subs="attributes+"]
 --
-NAME      HOST/PORT                                         PATH      SERVICES   PORT      TERMINATION
-jenkins   jenkins-{app-name}-{project-name}.{osl-route-hostname}             jenkins    <all>     edge/Redirect
+$ oc new-build {app-name}:latest~{scm-repo-url} \
+-e MAVEN_MIRROR_URL='http://nexus.{nexus-project-name}:8081/nexus/content/groups/public'
```

Comment:
I wonder why `8081` now, it wasn't required before?

Reply:
Not sure whether this is completely correct, but I based this procedure on a [similar procedure](https://docs.openshift.com/container-platform/3.5/dev_guide/app_tutorials/maven_tutorial.html) described in the Dev Guide for OCP 3.5. That procedure seems to require the port number, too.

Reply:
Oh, I see it now -- this is a URL of the service, not of the route. (As in, a cluster-internal URL, which isn't reachable from outside. Which is OK in this case.) And indeed the service exposes port `8081`.